src/pykeen/optimizers.py

```diff
@@ -52,30 +52,20 @@
 
 #: The default strategy for optimizing the optimizers' hyper-parameters (yo dawg)
 optimizers_hpo_defaults: Mapping[Type[Optimizer], Mapping[str, Any]] = {
-    Adadelta: dict(
```

Comment:
so the behavior for adadelta should be to use only the default and not try changing the learning rate?

Reply:
Exactly, see for example https://ruder.io/optimizing-gradient-descent/index.html#adadelta

Reply:
The relevant comment from there
> With Adadelta, we do not even need to set a default learning rate, as it has been eliminated from the update rule.